articles/how-to-insert-3d-objects-into-a-webpage-using-html-and-css/index.md

```diff
@@ -0,0 +1,320 @@
+
+## Create a webpage Strucure
+We are going to create a simple HTML structure. We can simplify the process using boilerplate generators which may come pre-installed by default in Visual Studio Code or as downloadable extensions by typing in the index file, "html:5" or "html5-boilerplate" then key in the Enter key. Since we don't need additional elements in the boilerplate generated, we will delete the extra code and link it to our CSS file. Let's also link it with some online scripts which will load some Font Awesome fonts and icons. The outcome will be as shown below:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>3D Object</title>
+    <script src='https://kit.fontawesome.com/a076d05399.js'></script>
+    <link rel="stylesheet" href="css/style.css">
+    
+</head>
+<body>
+    
+</body>
+</html>
+```
+Start the live server by either right-clicking on the "index.html" file in the explorer and selecting "Open with Live server" or on the status bar clicking on "Go live" to start the server. It will automatically open a preview of the webpage in the default browser which will be blank because we haven't added any content yet. I recommend you use chrome since it supports most 3D objects. You can change the default browser in the settings. If it doesn't automatically open the webpage preview in the browser, check the port number in which the server uses at the Visual Studio Code status bar then add access it in the browser by typing in the URL:
+
+~~~~
+http://127.0.0.1:5501/
+~~~~
+
+Port numbers vary from one to another. In my case, it is port number 5501. If the server doesn't launch completely, stop any services with a similar address running in the background. If it still has errors please look for additional support online concerning the error it displays.
+
+
+In the head section, add a link to import a model-viewer JavaScript code. This enables you to import the component into the webpage.
+
+```html
+    <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
+    <script nomodule src="https://unpkg.com/@google/model-viewer/dist/model-viewer-legacy.js"></script>
+```
+
+Copy your logo picture in the assets folder.
+When done, add a div element of id "container" to hold all our content in the body tags and also another in it for our navigation bar. Let us also add div elements with the following id; "aside" for holding our 3D object, "content" for some explanations on the object, and "icons" for some social media icons. Add a simple logo and some links to the webpage and some of their properties in css as shown below:
+
+```html
```

Comment:
If possible, add comments in the code snippet.

Reply:
Ok